src/java.base/share/classes/java/io/Console.java

```diff
@@ -86,7 +86,6 @@
  * char[] passwd;
  * if ((cons = System.console()) != null &&
  *     (passwd = cons.readPassword("[%s]", "Password:")) != null) {
- *     ...
```

Comment:
I assume you didn't mean to remove this, this is show that there is other code in this block before the passwd is cleared.